src/schemas/user.schema.ts

```diff
@@ -0,0 +1,11 @@
+import * as mongoose from 'mongoose';
+
+export const UserSchema = new mongoose.Schema({
+  first_name: String,
+  last_name: String,
+  email: String,
+  phone: String,
+  address: String,
+  description: String,
+  created_at: { type: Date, default: Date.now },
+});
```

Comment:
Create schema with using `nestjs/mongoose`
https://docs.nestjs.com/techniques/mongodb

Reply:
okay sure

Reply:
@Abhishek-kumar09 made this as per @nestjs/mongoose theme